locations/spiders/cvs.py

```diff
@@ -0,0 +1,83 @@
+import scrapy
+import re
+from locations.items import GeojsonPointItem
+
+class CVSSpider(scrapy.Spider):
+
+    name = "cvs"
+    allowed_domains = ["www.cvs.com"]
+    start_urls = (
+        'https://www.cvs.com/store-locator/cvs-pharmacy-locations',
+    )
+
+    def parse_hours(self, response):
+        store_title = response.xpath('normalize-space(//h4[@class="nomargin store-photo-heading"]/text())').extract_first()
+        hours = {}
+        if store_title:
+            lis = response.xpath('//ul[@class="cleanList srHours srSection"]/li')
+            hours [store_title] = {}
```

Comment:
Please remove the space between the variable name and the subscript operation.

Reply:
👍  sorry for this. Flake caught it but I forgot to put it. I also made the change here
```hours[store_title].update({day: times})]```